scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,58 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT current_database() || ''_admin''::text;'
```

Comment:
Yes, it is. I don't master delimiters at all so I copied an existing function, `cartodb.CDB_Organization_Member_Group_Role_Member_Name`. I've tested your approach and works like a charm, thank you very much!